packages/esm-appointments-app/src/appointments-calendar/calendar-patient-list/calendar-patient-list.component.tsx

```diff
@@ -25,8 +25,9 @@ interface CalendarPatientListProps {}
 const CalendarPatientList: React.FC<CalendarPatientListProps> = () => {
   const { t } = useTranslation();
   const currentPathName: string = decodeURI(window.location.pathname);
-  const serviceName = currentPathName.split('/')[5];
-  const forDate = currentPathName.split('/')[4];
+  const serviceName = currentPathName.split('/')[7];
+  const forDate = currentPathName.split('/')[6];
```

Comment:
I believe we can use the destructuring syntax here 
```suggestion
 const [, , , , , , forDate, serviceName] = currentPathName.split('/');
```

Reply:
Lets also add unit test for this to ensure its always working